app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -231,6 +232,11 @@ class HomeFragment : Fragment() {
             }
 
             view.tab_button.setCountWithAnimation(tabCount)
+            view.add_tabs_to_collections_button?.visibility = if (tabCount > 0) {
```

Comment:
The `?` is needed because pressing the new tab button in private mode crashes the app, presumable because `view.add_tabs_to_collections_button` doesn't exist on that screen.